forch/authenticator.py

```diff
@@ -99,6 +102,8 @@ def process_device_placement(self, src_mac, device_placement):
     def process_radius_result(self, src_mac, code, segment, role):
         """Process RADIUS result from radius_query"""
         LOGGER.info("Received RADIUS result: %s for src_mac: %s", code, src_mac)
+        if self._forch_metrics:
```

Comment:
is there any situation when this would not be defined?  Seems like unnecessary conditionals... ?

Reply:
Running the unit test, for example.